composer.json

```diff
@@ -47,6 +47,10 @@
 	"replace": {
 		"mediawiki/semantic-mediawiki": "*"
 	},
+	"suggest": {
```

Comment:
Why would we suggest those and not any other possible candidate it seems rather selective! I don't really mind it but SF seems a rather odd candidate given the fact that it no longer requires SMW.